src/main/java/picard/illumina/IlluminaBasecallsToSam.java

```diff
@@ -492,6 +506,10 @@ public static void main(final String[] args) {
             messages.add("The number of tags given in TAG_PER_MOLECULAR_INDEX does not match the number of molecular indexes in READ_STRUCTURE");
         }
 
+        if ((FIVE_PRIME_ADAPTER != null && THREE_PRIME_ADAPTER == null) || (THREE_PRIME_ADAPTER != null && FIVE_PRIME_ADAPTER == null)) {
+            messages.add("Either both or neither of THREE_PRIME_ADAPTER and FIVE_PRIME_ADAPTER must be set.");
```

Comment:
That sentence is one comma away from a lawsuit :-) 

http://www.npr.org/2017/03/23/521274657/the-10-million-lawsuit-that-hinges-on-an-oxford-comma

Could you reword so that it doesn't depend on the (lack of the) comma as much? 

How about "THREE_PRIME_ADAPTER and FIVE_PRIME_ADAPTER need to be both null or both set, but found to be differentially set."?